www/css/style.css

```diff
@@ -1,5 +1,29 @@
-/* Empty. Add your own CSS if you like */
+/* Scoped styles for Enketo */
+/* if we don't contain them here, they will leak into the rest of the app */
+.enketo-plugin {
+  @import 'enketo-core/src/sass/formhub/formhub.scss';
+}
+
+.enketo-plugin .form-header {
+  max-height: 50px;
+}
+
+.fill-container {
+  display: block;
+  position: relative;
+}
+
+.fill-container div[class*='css-view'] {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+}
 
+.fill-container div[class*='css-view'] > div[class*='css-view'] {
+  position: static;
+}
+
```

Comment:
For my own edification, these are adding fill-container to existing `div` styles.
But where are those div styles used? I have searched through the entire codebase and I don't see them. And I searched for css-view on the internet, but there don't seem to be any matches, so I am not sure it is standard terminology.

Reply:
It comes from React Native Web. The way it converts React Native markup into HTML is kind of messy. It adds a bunch of wrappers with randomized class names in order to apply the RN stylesheets as CSS.
Read here- https://giuseppegurgone.com/twitter-html
If you go to Twitter web and inspect, I think you'll see what I mean. The resulting HTML is not pretty, but it works.

In our case, React Native Web isn't really meant to be injected into an existing DOM structure. We need to control the behavior of the wrappers so they play nice with their AngularJS parents. To do this, I am selecting anything with a class that starts with `css-view`.

Once everything is rewritten we won't have to do this.